keel-core/src/main/kotlin/com/netflix/spinnaker/keel/IntentActivityRepository.kt

```diff
@@ -30,6 +33,24 @@ data class IntentActivity(
   val actor: String
 )
 
+/**
+ * @param intentId The ID of the intent that is being evaluated.
+ * @param changeType The type of change that took place this cycle.
+ * @param orchestrations A resultant (if any) list of orchestration IDs from the intent.
+ * @param messages Human-friendly messages about the change.
+ * @param diff The diff between current and desired state.
+ * @param actor Who (or what) initiated the operation.
+ */
+data class IntentConvergenceRecord(
+  val intentId: String,
+  val changeType: ChangeType,
+  val orchestrations: List<String>?,
+  val messages: List<String>?,
+  val diff: Set<FieldState>,
+  val actor: String,
+  val timeMs: Long
```

Comment:
If we're going full on javadoc ... do we want to include a definition of `timeMs`?

Reply:
I'll change to timestampMillis also following our discussion on unit naming